### ISSUE TITLE
Add XML importing + exporting!

### ISSUE THREAD

**Issue by Flohhhhh** [Quboid/CS2-MoveIt#2]
This was one of my favorite features in CS1, I love to spend lots of time on interchanges and re-use them.

Would be cool if we could paste xml from outside the game as well, or at least create files that would be loaded in game

**Comment by Quboid** [comment]
This will be added but it will take a lot of time as it depends on other features, in particular copying.

**Comment by Flohhhhh** [comment]
Of course, glad to know it's in the roadmap, I'll consider learning some Unity to help out at some point 🫡
